docker-compose.yml

```diff
@@ -0,0 +1,61 @@
+version: '3.3'
+
+x-db-settings: &db-settings
```

Comment:
I think this has the same issue as localdev did in that it has too many things in it i.e. env vars and other properties. Just also realised that it is only used once so it might be more readable to not make it a "variable" and just use it inline in the service. Then the home and maven_user_home wont look out of place in db-settings

Reply:
sure, got it fixed locally. Will push ASAP.